<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Układ obrazów</string>
  </property>
  <property name="windowIcon">
   <iconset theme="video-display">
    <normaloff/>
   </iconset>
  </property>
  <layout class="QGridLayout" name="layoutMain">
   <item row="0" column="0" rowspan="5">
    <widget class="QGroupBox" name="groupSetup">
     <property name="title">
      <string>Ekran</string>
     </property>
     <layout class="QVBoxLayout" name="layoutSetup">
      <item>
       <widget class="QRadioButton" name="setupMaster">
        <property name="text">
         <string>Główny</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="setupSlave">
        <property name="text">
         <string>Dodatkowy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="setupBoth">
        <property name="text">
         <string>Osobne ekrany</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="setupClone">
        <property name="text">
         <string>Klonuj ekrany</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="Cancel">
     <property name="text">
      <string>Anuluj</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="Save">
     <property name="text">
      <string>Wybierz</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QGroupBox" name="groupDevice">
     <property name="title">
      <string>Wybór urządzeń</string>
     </property>
     <layout class="QFormLayout" name="layoutDevice">
      <item row="0" column="0">
       <widget class="QLabel" name="labelMaster">
        <property name="text">
         <string>Podstawowe</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="Master"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelSlave">
        <property name="text">
         <string>Dodatkowe</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="Slave"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QGroupBox" name="groupResolution">
     <property name="title">
      <string>Rozdzielczość klonowania</string>
     </property>
     <layout class="QGridLayout" name="layoutResolution">
      <item row="0" column="0">
       <widget class="QComboBox" name="Resolution">
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <item>
         <property name="text">
          <string>800x600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1024x768</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1366x768</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>setupMaster</tabstop>
  <tabstop>setupSlave</tabstop>
  <tabstop>setupBoth</tabstop>
  <tabstop>setupClone</tabstop>
  <tabstop>Master</tabstop>
  <tabstop>Slave</tabstop>
  <tabstop>Resolution</tabstop>
  <tabstop>Save</tabstop>
  <tabstop>Cancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>Cancel</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Save</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
